### ISSUE TITLE
Add Home Screen with Podcasts and mock Subscriptions

### ISSUE THREAD

**Issue by shrirambalaji** [cast-bucket/cast-bucket#4]

**Comment by jashgala** [comment]
Do we have an API source in mind which can provide us with the top podcasts?

**Comment by shrirambalaji** [comment]
Actually no, the db itself has been populated based on data from multiple resources.
We'd need a way to do some more crawling and identify the most subscribed podcasts.

**Comment by jashgala** [comment]
Perhaps we can use some API like ListenNotes: https://www.listennotes.com/api/
Other links from this StackOverflow question: https://stackoverflow.com/questions/3201052/podcast-search-api

I believe crawling and scraping will not be straight forward.

**Comment by shrirambalaji** [comment]
I tried a couple of options like listennotes and [gpodder](https://gpodder.net) and decided against using their api, due to multiple issues. As a matter of fact, I went so far as to actually create a node library for gpodder called [node-gpodder](https://github.com/Shriram-Balaji/node-gpodder) but, the api wasn't updated frequently and it was a cumbersome process to integrate with the frontend.

I have actually crawled the data for podcasts, and the parsed data is now available in firebase. One of the sources for the data is this [amazing list](https://github.com/rShetty/awesome-podcasts). 

The task at hand, would be to identify which podcasts have how many subscribers, but I guess we can handle Top Podcasts later. **Edit**: Updated issue to use a mock subscriptions in the home page.
